hnn_core/cell.py

```diff
@@ -655,13 +655,20 @@ def parconnect_from_src(self, gid_presyn, nc_dict, postsyn,
 
         return nc
 
-    def plot_morphology(self, ax=None, cell_types=None, show=True):
```

Comment:
Also I don't think `cell_types` is used anywhere? @jasmainak

Reply:
I thought this was a method of `Network` object so you can loop over the `cell_types` in a network objec ... I remember a student was working on this but not sure if it ever got merged

Reply:
Yeah I think that ended up getting dropped, the tests are passing so I think it's safe to remove for now